packages/tooltip/src/Tooltip.story.js

```diff
@@ -1,137 +1,249 @@
-    >
-      <div style={{ position: "relative", width: "50px", border: "1px solid" }}>
-        Child
+
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          width: "100%",
+          position: "fixed",
+          bottom: 0,
+        }}
+      >
         <Tooltip
-          align={alignments.AUTO}
-          text="This is an auto aligned tooltip"
-          bondariesSelector=".exampleContainer"
+          trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+          content="Auto aligned/positioned tooltip"
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
         />
-      </div>
-      <div style={{ position: "relative", width: "50px", border: "1px solid" }}>
-        Child
         <Tooltip
-          align={alignments.AUTO}
-          text="This is an auto aligned tooltip"
-          bondariesSelector=".exampleContainer"
+          trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+          content="Auto aligned/positioned tooltip"
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
         />
       </div>
     </div>
   ))
-  .add("custom padding", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip
-        isVisible
-        text="Left aligned with custom padding"
-        padding="30px"
-      />
-    </div>
-  ))
-  .add("custom top", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip isVisible text="Left aligned with custom top" top="30vh" />
-    </div>
-  ))
-  .add("with content overflow hidden", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip isVisible padding="0" overflow="hidden">
-        <div style={{ background: "aliceblue" }}>It should not overflow</div>
-      </Tooltip>
-    </div>
-  ))
-  .add("extended style", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip
-        css="
-          text-decoration-line: underline overline;
-        "
-        isVisible
-        text="This is a left aligned tooltip"
-      />
-    </div>
-  ));
+  .add("auto aligned/positioned (with boundaries)", () => {
+    // eslint-disable-next-line react/prop-types
+    const BoundaryContainer = ({ children, style, ...props }) => (
+      <div
+        style={{
+          border: "solid 1px orangered",
+          display: "flex",
+          justifyContent: "space-between",
+          width: "600px",
+          ...style,
+        }}
+        {...props}
+      >
+        {children}
+      </div>
+    );
+    return (
+      <div>
+        <BoundaryContainer className="exampleContainer">
+          <Tooltip
+            trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+            content="Auto aligned/positioned tooltip"
+            positionX={POSITIONS.X.AUTO}
```

Comment:
I think that we can simplify the position API, something like `placement`: `top`. 'topRight', 'bottomLeft'. What do you think?

Reply:
I see that some libs are going that way, but for our use case I personally prefer the way it was implemented because we can set the position on each axis without considering the position of the other one.
We would have to make some tweaks to work the way you proposed.

What's your opinion @vNakamura?

Reply:
I'm good with two props.
In most cases, we would just set `positionX`. We never needed the tooltip at the top until now.

Reply:
Ok, I prefer an enum of string that covers all the cases, easier to use and read. But if this way is ok for you guys, I'm ok too!